lib/ansible/modules/cloud/amazon/rds_snapshot.py

```diff
@@ -59,29 +72,50 @@
       - whether to remove tags not present in the C(tags) parameter.
     default: True
     type: bool
+  fail_on_not_exists:
```

Comment:
Is this a genuine problem? This is a very unusual way of doing things (you could always use rds_snapshot_info and then fail if it doesn't exist)